src/utils/unmarkedWinterStorage.ts

```diff
@@ -1,7 +1,8 @@
 import { List } from 'immutable';
+import _ from 'lodash';
```

Comment:
For a smaller bundle size:
```suggestion
import orderBy from 'lodash/orderBy';
```